.githooks/pre-commit

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
```

Comment:
run `shellcheck` against this script and if any issues are reported by it, correct them.  Then let me know here when `shellcheck` is running clean.